src/main/java/servlets/VendorServlet.java

```diff
@@ -14,20 +14,35 @@
 
 package servlets;
 
+import data.FileParser;
+
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 @WebServlet("/VendorServlet")
 public class VendorServlet extends HttpServlet {
   @Override
+  /**
+   * Prints a hashmap to the response object
+   * STRING (Vendor ID) : ARRAY OF STRINGS (Account IDs)
+   * e.g., {"vend_1":["account1","account2","account3","account4"],
+   *            "vend_2":["account5","account6","account7"}
+   */
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // GET method --> returns a JSON array of existing Vendor IDs
-    String dummyResponse = "[\"CADE01\",\"VINCENT01\",\"CHARLIE01\",\"JAKE01\",\"IAN01\"]";
+    FileParser fileParser = new FileParser();
+    HashMap<String, ArrayList<String>> configMap = fileParser.getConfigMap();
```

Comment:
Maybe instead of returning a HashMap, you could just return the JSON String. I feel that this doGet would be cleaner if you did it that way because you're not really doing anything with the HashMap in this method.

Reply:
got it. good idea!